test/aws.deployer.test.js

```diff
@@ -13,12 +13,25 @@
 /* eslint-env mocha */
 import assert from 'assert';
 import nock from 'nock';
+import xml2js from 'xml2js';
 import BaseConfig from '../src/BaseConfig.js';
 import AWSConfig from '../src/deploy/AWSConfig.js';
 import AWSDeployer from '../src/deploy/AWSDeployer.js';
 import ActionBuilder from '../src/ActionBuilder.js';
 
 describe('AWS Deployer Test', () => {
+  beforeEach(() => {
+    process.env.AWS_ACCESS_KEY_ID = 'fake';
+    process.env.AWS_SECRET_ACCESS_KEY = 'fake';
+    process.env.AWS_REGION = 'us-east-1';
```

Comment:
otherwise leads to errors in tests that invoke `GetCallerIdentity` (Missing credentials)